docs/guides/writingafunction.md

```diff
@@ -1,51 +1,84 @@
 # Writing a Function
 
-TriggerMesh Function CR provides ample opportunities to implement custom events
-flow logic and can act as a source, transformation, or target. Currently,
-Functions supports Python, NodeJS, and Ruby runtimes.
+The TriggerMesh `Function` API provides opportunities to implement custom event flow logic and may act as a source, transformation, or a target. Currently, Python, NodeJS, and Ruby runtimes are supported.
 
-## Prerequisites
+!!! Info "Prerequisites"
+    You need a working TriggerMesh platform installation. See the [installation steps](installation.md).
+    You can verify that the API is available with the following command:
 
-1. K8s cluster and configured kubectl
-1. [Knative Serving and Eventing Operators](https://knative.dev/docs/admin/install/knative-with-operators/)
-1. [TriggerMesh Function](https://github.com/triggermesh/function)
+    ```console
+    $ kubectl get crd |grep function
+    functions.extensions.triggermesh.io        2021-10-06T09:01:33Z
+    ```
```

Comment:
grep is totally unnecessary here.

```suggestion
    ```console
    $ kubectl get crd functions.extensions.triggermesh.io
    NAME                                  CREATED AT
    functions.extensions.triggermesh.io   2021-10-06T09:01:33Z
    ```
```